subworkflows/local/starfusion_workflow/tests/main.nf.test.snap

```diff
@@ -0,0 +1,38 @@
+{
+    "STARFUSION_WORKFLOW - Homo sapiens - FASTQs minigenome": {
+        "content": [
+            [
+                [
+                    {
+                        "id": "test_fastqs"
+                    },
+                    "test_fastqs.starfusion.fusion_predictions.tsv:md5,abe17134a231642edf9351e4964e8a97"
+                ]
+            ],
+            "test_fastqs.Log.final.out",
+            [
+                [
+                    {
+                        "id": "test_fastqs"
+                    },
+                    "test_fastqs.ReadsPerGene.out.tab:md5,8e0d42deeea09924d5c7ba3147bbfd78"
+                ]
+            ],
+            [
+                
+            ],
+            [
+                
+            ],
+            [
+                "versions.yml:md5,afde4f2fd6056df81e322b3c35ab7a8a",
```

Comment:
are these empty ouputs expected?

Reply:
these are not expected but I added a TODO because I think that this output is not being produced from STAR